sycl/doc/extensions/experimental/sycl_ext_oneapi_device_architecture.asciidoc

```diff
@@ -295,6 +337,167 @@ of these enumerators, and it provides a brief description of their meanings.
 |`intel_gpu_12_10_0`
 |1
 |Alias for `intel_gpu_dg1`.
+
+|`nvidia_gpu_sm20`
+|1
+|NVIDIA Fermi architecture.
+
+|`nvidia_gpu_sm30`
+|1
+|NVIDIA Kepler architecture.
+
+|`nvidia_gpu_sm32`
+|1
+|NVIDIA Kepler architecture.
```

Comment:
Listing the same architecture here is confusing, and I suspect some readers will struggle to understand the difference between the sm30, sm32, sm35 and sm37 lines in the table.

Why not list the compute capability too, to make it obvious that's what the number means? e.g.

- `nvidia_gpu_sm30`, NVIDIA Kepler architecture (compute capability 3.0)
- `nvidia_gpu_sm32`, NVIDIA Kepler architecture (compute capability 3.2)

I think it might also be a good idea to add a non-normative note above or below the table that says something like:

"For NVIDIA GPUs, the architecture enumerator corresponds to the [compute capability](https://docs.nvidia.com/cuda/cuda-c-programming-guide/index.html#compute-capabilities) of the device, and `ext_oneapi_architecture_is` can be used similarly to the `__CUDA_ARCH__` macro in CUDA."

...since it may help folks migrating from CUDA to SYCL.

Reply:
thanks @Pennycook for review. I applied your comments in.